api/tests/integration/migrations/extract-person-contacts.js

```diff
@@ -1,3 +1,13 @@
+// Because we're doing DB switching we need invalidate the cache so the require
+// gets processed again
+[
+  'migrations/extract-person-contacts.js',
+  'controllers/people.js',
+  'controllers/places.js'
+].forEach(k => {
+  delete require.cache[require.resolve('../../../' + k)];
+});
+
```

Comment:
I think if you rearchitect the lineage util to not have globals this goes away (comments below).